### ISSUE TITLE
How to get the depth map of the test set？

### ISSUE THREAD

**Issue by xy1999729** [DNMP/DNMP#18]
Thanks for your inspiring work. When I run the training script for the waymo dataset, there was an error about missing the depth map of the validation set. Should I generate the depth map by mixing the train and validate set and run colmap or just run the colmap for the validate set?

**Comment by FanLu97** [comment]
Hi, we do not need depth map on validation set. You can simply add a filter in `waymo_dataset.py` to skip that in validation (simple example in [code](https://github.com/DNMP/DNMP/blob/ff404e2a95b09fda450fce8d52cd528a5c51100a/datasets/kitti360_dataset.py#L36C1-L43C32)).
